### ISSUE TITLE
Some packages won't install under Solaris 7

### ISSUE THREAD

**Issue by vkoskiv** [tgc/tgcwarev2-for-solaris#2]
Hello! First off, really cool of you to maintain this project for older versions of Solaris. I'm getting acquainted with my first SPARC system, and it's nice to have a more familiar userspace to use with Solaris 7.

I grabbed the packages from `http://jupiterrise.com/tgcware/sunos5.7_sparc/stable/` and most of these work just fine, but I noticed that these packages don't seem to install:

- TGCXlgomp1
- TGCXlibcloog0
- TGCXlibppl11
- TGCXlibpplc4

There may be more that don't work, but these popped up as the first dependencies for the things I was installing. Namely, GCC.

The error is similar on all of them:
```
bash-4.4# pkgadd -d libcloog0-0.15.11-1.tgc-sunos5.7-sparc-tgcware
pkgadd: ERROR: no packages were found in </var/tmp/dstreAAAaSaaUb>
```

I browsed [various](https://www.unix.com/solaris/259753-pkgadd-error-no-packages-found-var.html) [forums](https://www.tek-tips.com/viewthread.cfm?qid=1233725) and tried some suggestions, but none seem to work:

```
bash-4.4# cp libgomp1-4.5.4-3.tgc-sunos5.7-sparc64-tgcware64 /var/spool/pkg/
bash-4.4# pkgadd libgomp1-4.5.4-3.tgc-sunos5.7-sparc64-tgcware64 
pkgadd: ERROR: no packages were found in </var/spool/pkg>
bash-4.4#
```

```
bash-4.4# pkgadd -d /export/home/temp/libgomp1-4.5.4-3.tgc-sunos5.7-sparc64-tgcware64 
pkgadd: ERROR: no packages were found in </var/tmp/dstreAAAdea4Xb>
```

If there is anything I could do to help debug this, I'd be happy to. Everything else has installed just fine with `pkgadd -d <pkg_file_name>`

Here is my system configuration, just a freshly installed Solaris 7 11/99 on a Sun Ultra 5:
```
bash-4.4# prtconf 
System Configuration:  Sun Microsystems  sun4u
Memory size: 128 Megabytes
System Peripherals (Software Nodes):

SUNW,Ultra-5_10
    packages (driver not attached)
        terminal-emulator (driver not attached)
        deblocker (driver not attached)
        obp-tftp (driver not attached)
        disk-label (driver not attached)
        SUNW,builtin-drivers (driver not attached)
        sun-keyboard (driver not attached)
        ufs-file-system (driver not attached)
    chosen (driver not attached)
    openprom (driver not attached)
        client-services (driver not attached)
    options, instance #0
    aliases (driver not attached)
    memory (driver not attached)
    virtual-memory (driver not attached)
    pci, instance #0
        pci, instance #0
            ebus, instance #0
                auxio (driver not attached)
                power, instance #0
                SUNW,pll (driver not attached)
                se, instance #0
                su, instance #0
                su, instance #1
                ecpp (driver not attached)
                fdthree, instance #0
                eeprom (driver not attached)
                flashprom (driver not attached)
                SUNW,CS4231 (driver not attached)
            network, instance #0
            SUNW,m64B, instance #0
            ide, instance #0
                disk (driver not attached)
                cdrom (driver not attached)
                dad, instance #0
                sd, instance #0
        pci, instance #1
            pci (driver not attached)
                pci108e,1000 (driver not attached)
                SUNW,qfe (driver not attached)
                pci108e,1000 (driver not attached)
                SUNW,qfe (driver not attached)
                pci108e,1000 (driver not attached)
                SUNW,qfe (driver not attached)
                pci108e,1000 (driver not attached)
                SUNW,qfe (driver not attached)
    SUNW,UltraSPARC-IIi (driver not attached)
    pseudo, instance #0
bash-4.4# uname -a                                                                                                                                                                                                                                                                         
SunOS ultra5 5.7 Generic_106541-08 sun4u sparc SUNW,Ultra-5_10
bash-4.4# showrev -p                                                                                                         
Patch: 107648-09 Obsoletes:  Requires: 107078-17 Incompatibles:  Packages: SUNWxwrtx, SUNWxwrtl, SUNWxwice, SUNWxwplt, SUNWxwicx, SUNWxwplx, SUNWxwinc, SUNWxwman, SUNWxwpmn, SUNWxwslb, SUNWxwslx
Patch: 107656-05 Obsoletes:  Requires: 107648-06 Incompatibles:  Packages: SUNWxwrtx, SUNWxwrtl, SUNWxwplt, SUNWxwplx, SUNWxwpmn, SUNWxwslb, SUNWxwslx
Patch: 107430-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWwsr
Patch: 108029-02 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWwsr
Patch: 107063-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWtleux
Patch: 107437-03 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWtiu8x, SUNWtiu8
Patch: 107316-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWploc, SUNWplcx
Patch: 106541-08 Obsoletes: 106832-03, 106976-01, 107029-01, 107030-01, 107334-01, 107031-01, 107117-05 Requires: 107544-02 Incompatibles:  Packages: SUNWkvmx, SUNWkvm, SUNWcsu, SUNWcsr, SUNWcslx, SUNWcsl, SUNWcarx, SUNWcar, SUNWesu, SUNWarc, SUNWarcx, SUNWatfsr, SUNWscpu, SUNWcpr, SUNWcprx, SUNWcsxu, SUNWdpl, SUNWdplx, SUNWesxu, SUNWhea, SUNWipc, SUNWtoo, SUNWpcmci, SUNWpcmcu, SUNWpcmcx, SUNWtnfc, SUNWtnfcx, SUNWtoox, SUNWvolr
Patch: 107453-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWkvm, SUNWcar
Patch: 107003-03 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWi8rf
Patch: 107962-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWhiu8x, SUNWhiu8, SUNWciu8x, SUNWciu8
Patch: 107917-02 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWeuise
Patch: 107738-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWeuise, SUNWploc1, SUNWplow1, SUNWpldte
Patch: 106857-10 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWeuise, SUNWplow, SUNWplow1, SUNWpldte
Patch: 106793-03 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsu, SUNWhea
Patch: 106940-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsu
Patch: 106950-07 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsu, SUNWcslx, SUNWcsl, SUNWarc, SUNWbtool, SUNWbtoox, SUNWcsxu, SUNWhea, SUNWtoo, SUNWtoox
Patch: 106985-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsu, SUNWcsr
Patch: 106987-02 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsu, SUNWsutl
Patch: 107018-01 Obsoletes:  Requires: 106938-01 Incompatibles:  Packages: SUNWcsu
Patch: 107285-02 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsu, SUNWcslx, SUNWcsl, SUNWnisu
Patch: 107401-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsu
Patch: 107403-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsu, SUNWcsxu
Patch: 107441-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsu
Patch: 107443-09 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsu
Patch: 107448-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsu
Patch: 107454-03 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsu
Patch: 107474-01 Obsoletes:  Requires: 107292-02 Incompatibles:  Packages: SUNWcsu, SUNWcsxu
Patch: 107475-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsu
Patch: 107477-02 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsu
Patch: 107544-03 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsu, SUNWcsr
Patch: 107551-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsu, SUNWxcu4
Patch: 107624-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsu
Patch: 107744-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsu, SUNWxcu4
Patch: 107792-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsu
Patch: 107836-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsu
Patch: 107843-02 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsu, SUNWcsr
Patch: 108089-02 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsu, SUNWxcu4
Patch: 108148-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsu, SUNWcsxu
Patch: 108158-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsu
Patch: 108162-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsu, SUNWcsr
Patch: 108203-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsu, SUNWcsxu, SUNWhea
Patch: 107458-04 Obsoletes: 107121-02 Requires: 107834-02 Incompatibles:  Packages: SUNWcsu, SUNWcsr, SUNWcarx, SUNWssad
Patch: 107451-02 Obsoletes:  Requires: 107117-03 Incompatibles:  Packages: SUNWcsu
Patch: 107589-03 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsu, SUNWcsr, SUNWcarx, SUNWcar, SUNWhea, SUNWpd, SUNWpdx
Patch: 106924-02 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsr, SUNWcarx
Patch: 106936-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsr
Patch: 106944-02 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsr, SUNWcarx
Patch: 106948-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsr, SUNWcarx
Patch: 106963-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsr, SUNWcarx
Patch: 107148-04 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsr, SUNWcarx
Patch: 107456-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsr
Patch: 107459-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsr, SUNWcarx
Patch: 107460-03 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsr, SUNWcarx, SUNWhea
Patch: 107462-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsr, SUNWcarx
Patch: 107465-02 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsr, SUNWcarx
Patch: 107680-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsr, SUNWcarx
Patch: 107796-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsr, SUNWcarx
Patch: 107841-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsr, SUNWcarx
Patch: 107865-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsr, SUNWcarx, SUNWhea
Patch: 107809-03 Obsoletes:  Requires: 106541-08 Incompatibles:  Packages: SUNWcsr, SUNWcsd
Patch: 108285-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcsr
Patch: 106938-02 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcslx, SUNWcsl
Patch: 106942-05 Obsoletes: 107215-01 Requires:  Incompatibles:  Packages: SUNWcslx, SUNWcsl, SUNWarc, SUNWnisu
Patch: 106980-07 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcslx, SUNWcsl, SUNWhea
Patch: 107332-02 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcslx, SUNWcsl, SUNWarc
Patch: 108244-01 Obsoletes:  Requires: 106541-08 Incompatibles:  Packages: SUNWcslx, SUNWcsl
Patch: 106917-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWciu8x, SUNWciu8
Patch: 107147-05 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcarx, SUNWcar
Patch: 108224-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWcarx, SUNWcar
Patch: 107078-18 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWxwfnt, SUNWxwplt, SUNWxwslb
Patch: 107081-08 Obsoletes: 107607-01, 107802-01 Requires:  Incompatibles:  Packages: SUNWmfrun, SUNWdtbax
Patch: 107059-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWesu, SUNWxcu4
Patch: 107799-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWesu
Patch: 107893-02 Obsoletes: 108237-01, 108122-01 Requires:  Incompatibles:  Packages: SUNWtltk, SUNWtltkx
Patch: 107014-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWxilow
Patch: 107250-02 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWxwplt
Patch: 107650-05 Obsoletes:  Requires: 107648-06 Incompatibles:  Packages: SUNWxwplt, SUNWxwpmn, SUNWxwslb
Patch: 107652-05 Obsoletes:  Requires: 107648-06 Incompatibles:  Packages: SUNWxwplt, SUNWxwman, SUNWxwpmn
Patch: 107654-05 Obsoletes:  Requires: 107648-08 Incompatibles:  Packages: SUNWxwplt, SUNWxwman
Patch: 107807-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWxwplt
Patch: 107233-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWxwopt
Patch: 107853-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWxwopt
Patch: 106934-03 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWdtbas
Patch: 107094-04 Obsoletes:  Requires: 107081-07 Incompatibles:  Packages: SUNWdtbas
Patch: 107178-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWdtbas
Patch: 108219-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWdtbas
Patch: 107881-10 Obsoletes:  Requires: 107200-08, 107226-07, 107248-01, 108151-01, 108343-01 Incompatibles:  Packages: SUNWdtbas, SUNWdtdte, SUNWdtbax, SUNWdtdst, SUNWdtezt, SUNWdtinc, SUNWdtmad, SUNWdtwm
Patch: 106946-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWaccu
Patch: 107557-02 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWaccu
Patch: 107587-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWaccu
Patch: 106978-09 Obsoletes:  Requires: 107456-01 Incompatibles:  Packages: SUNWadmap, SUNWadmc
Patch: 108170-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWadmc
Patch: 108175-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWadmc
Patch: 106148-05 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWxfb, SUNWxfbx
Patch: 107351-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWxglrt, SUNWxgldg
Patch: 107553-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWpppk, SUNWpppkx
Patch: 107794-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWast
Patch: 107127-02 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWatfsu
Patch: 107001-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWdtdte
Patch: 107180-12 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWdtdte
Patch: 107887-07 Obsoletes:  Requires: 107001-01 Incompatibles:  Packages: SUNWdtdte, SUNWdtdst, SUNWdtma
Patch: 107022-04 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWdtdmn
Patch: 108221-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWdtdmn
Patch: 106949-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWbcp
Patch: 106952-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWbnuu
Patch: 107038-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWdoc
Patch: 107011-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWdtdst
Patch: 107072-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWdtdst
Patch: 107200-08 Obsoletes:  Requires: 107001-01 Incompatibles:  Packages: SUNWdtdst
Patch: 107219-02 Obsoletes:  Requires: 106934-02 Incompatibles:  Packages: SUNWdtdst
Patch: 107248-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWdtdst, SUNWdtma
Patch: 107306-01 Obsoletes:  Requires: 106934-03 Incompatibles:  Packages: SUNWdtdst
Patch: 107688-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWdtdst
Patch: 107711-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWdtdst, SUNWdtma
Patch: 107885-05 Obsoletes:  Requires: 107219-02 Incompatibles:  Packages: SUNWdtdst, SUNWdthev
Patch: 108197-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWdtdst
Patch: 108151-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWdtezt
Patch: 108343-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWdtezt
Patch: 107883-04 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWdtim
Patch: 107226-07 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWdtwm
Patch: 107702-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWdtwm
Patch: 106999-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWesxu
Patch: 107472-02 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWses
Patch: 106327-05 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWlibC
Patch: 107293-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWgss, SUNWgssx
Patch: 107834-02 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWhea
Patch: 107919-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWhea
Patch: 106982-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWhmd, SUNWhmdx
Patch: 108263-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWhmd, SUNWhmdx
Patch: 107473-02 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWluxop
Patch: 107074-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWinst
Patch: 107432-03 Obsoletes: 106812-04 Requires:  Incompatibles:  Packages: SUNWisolc, SUNWisolx, SUNWploc1, SUNWplc1x, SUNWplow1
Patch: 107813-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWjiu8, SUNWjiu8x
Patch: 107337-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWkcsrt, SUNWkcspg
Patch: 107555-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWldapx, SUNWlldap
Patch: 107469-03 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWluxd, SUNWluxl
Patch: 106960-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWman
Patch: 108068-03 Obsoletes: 106851-04, 107318-04, 107784-03 Requires:  Incompatibles:  Packages: SUNWman
Patch: 106146-09 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWm64, SUNWm64cf, SUNWm64w, SUNWm64x
Patch: 107709-03 Obsoletes: 106788-02, 107445-03 Requires:  Incompatibles:  Packages: SUNWmibii, SUNWsasnm, SUNWsadmi, SUNWsasnx
Patch: 107330-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWntpu
Patch: 107115-03 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWpcu, SUNWpsu
Patch: 106925-02 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWpd, SUNWpdx
Patch: 107185-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWploc1, SUNWplow1
Patch: 107187-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWploc1, SUNWplow1
Patch: 107746-03 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWploc1
Patch: 107438-02 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWplow
Patch: 107044-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWplow1
Patch: 107499-02 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWpldte
Patch: 106879-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWpmowu
Patch: 106887-02 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWrtvcl
Patch: 107359-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWscbcp
Patch: 107684-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWsndmu
Patch: 107058-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWsprot
Patch: 107972-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWsutl
Patch: 107171-04 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWswmt
Patch: 107049-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWtddte
Patch: 107838-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWtnfc, SUNWtnfcx
Patch: 107076-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWvolu, SUNWvolux
Patch: 107259-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWvolu
Patch: 107584-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWvolu
Patch: 108147-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWxilcg
Patch: 106147-04 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWxilvl
Patch: 108168-01 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWxwinc
Patch: 107658-04 Obsoletes:  Requires: 107648-05 Incompatibles:  Packages: SUNWxwpmn
Patch: 107546-02 Obsoletes:  Requires:  Incompatibles:  Packages: SUNWxwpsr
```

**Comment by tgc** [comment]
The build host has more recent patches and crucially it has patches for all the package utilities installed.
The newer versions can handle PKGNAMEs longer than 8 characters which is required for a little subset of the tgcware packages.
You can find showrev output from the build host on my website to see the exact revisions the packages have been built with.

Below are the patch numbers for the package utilities on Solaris 7/SPARC. I suspect 107171 might be the one you really need but you should try and make sure you have all of them installed.

107332
107443
111113
108162
112590
107171

**Comment by vkoskiv** [comment]
Yep, installing patches resolved the issue for me. I don't happen to have a support contract with Oracle, but I found the recommended Solaris 7 patches hosted by some very kind people [here](https://ftp.zx.net.nz/pub/Patches/sun/solaris/clusters/7_Recommended.zip). I installed all of them to be certain, and everything was smooth sailing from there.

Thanks again for maintaining these packages, they have already made my Solaris+UltraSPARC adventure much more enjoyable!
